docs/ota-client-guide/modules/ROOT/pages/ecu_events.adoc

```diff
@@ -7,6 +7,8 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 ====
 endif::[]
 
+// MC: This page is currently hidden in the developer guide portal and will be updated later to be connected to the API documentation.
```

Comment:
@ammaboateng: "MC" are the initials of a previous tech writer. If you want to leave a note like this, put your own initials on it.